cypress/e2e/course_board/openDrawingElementOnBoard.feature

```diff
@@ -1,25 +1,71 @@
 @pr
 @release
+@stable_test
+@api-migrated
 Feature: Course Board - Opening a drawing element on a course page
 
     As a teacher and student I want to open already created drawing element on a course page
 
-    @stable_test
-    Scenario Outline:  Teacher is able to open a drawing element to the course board
+    Scenario Outline: Users (admin, teacher, student), create a course with board and open TLDraw.
+        Given I am logged in as a '<admin>' at '<namespace>'
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        Given I am logged in as a '<student>' at '<namespace>'
 
-        # Teacher open the drawing element to the course column
-        Given I am logged in as a 'teacher1_nbc' at 'nbc'
+        # Pre-Condition: Creating new course
```

Comment:
this should be --> # pre-condition: admin creates a course and assign teacher to the course